.github/workflows/rgw-multisite-test/action.yml

```diff
@@ -82,9 +82,9 @@ runs:
         tests/scripts/github-action-helper.sh verify_operator_log_message "${committed_msg} ${ns_name_primary}"
         tests/scripts/github-action-helper.sh verify_operator_log_message "${committed_msg} ${ns_name_secondary}"
         tests/scripts/github-action-helper.sh restart_operator
-        not_committed_msg="there are no changes to commit for RGW configuration period for CephObjectStore"
-        tests/scripts/github-action-helper.sh wait_for_operator_log_message "${not_committed_msg} ${ns_name_primary}" 120
-        tests/scripts/github-action-helper.sh wait_for_operator_log_message "${not_committed_msg} ${ns_name_secondary}" 90
+#        not_committed_msg="there are no changes to commit for RGW configuration period for CephObjectStore"
+#        tests/scripts/github-action-helper.sh wait_for_operator_log_message "${not_committed_msg} ${ns_name_primary}" 120
+#        tests/scripts/github-action-helper.sh wait_for_operator_log_message "${not_committed_msg} ${ns_name_secondary}" 90
 
```

Comment:
@BlaineEXE : I have commented following case, since with this change `JoinMultisite` will be called only if Endpoint is missing in the list, so `commitConfigChanges()` won't be reached during normal operator pod restart. May I remove this test case?

Reply:
I thought this was ensuring the period should be committed to the RGW. This ensures the RGW got updated as appropriate. Shouldn't the period still be updated when the zone is configured? Should we instead change the test to check for the zone update?

Reply:
`commitConfigChanges` is called from `removeObjectStoreFromMultisite`, `JoinMultisite` [mulitsite related] and `createMultisite` in object-store-controller(not zone-controller). The `removeObjectStoreMultisite` is called only on deletion of objectstore and earlier `JoinMultisite` for each object store reconcile, now I change that behaviour `JoinMulitiste` will be called only if rgw endpoint is not present in the zone endpoint list

Reply:
Does the period need updated when a zone's endpoints change?

Reply:
yes, this is also how endpoints/rgw can be removed a zone in multisite